backend/src/main/java/ca/bc/gov/hlth/hnweb/service/EligibilityService.java

```diff
@@ -27,4 +65,65 @@ public CheckEligibilityResponse checkEligibility(String phn, Date eligibilityDat
 		return checkEligibilityResponse;
 	}
 
+	public Message checkMspCoverageStatus(E45 e45, String transactionId) throws HL7Exception {		
+		
+		String e45v2 = encodeE45ToV2(e45);		
```

Comment:
Does this need to be a seperate method that's specific to E45 or should we just call `parser.encode(e45)` right here

Reply:
Didn't need separate method, removed.